### ISSUE TITLE
Axios vulnerability

### ISSUE THREAD

**Issue by vmashnitskaya** [serverless/serverless#12232]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest v3 release?

- [X] Yes, I'm using the latest v3 release

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

The serverless has a vulnerable axios version in package @serverless/platform-client. Is it possible to update it to the latest one?

### Service configuration (serverless.yml) content

```yaml
N/A
```

### Command name and used flags

N/A

### Command output

```shell
N/A
```

### Environment information

```shell
Framework Core: 3.35.2 (local)
Plugin: 7.0.5
SDK: 4.4.0
```

**Comment by jnovek** [comment]
This is the report snyk gives us.

If upgrading quickly isn't an option (it's a major version, 0.21.4 --> 1.6.0), there is also a workaround mentioned in the snyk report.

https://security.snyk.io/vuln/SNYK-JS-AXIOS-6032459

**Comment by samwaterlily** [comment]
Is there any work being done on addressing this?

**Comment by psinglet** [comment]
We need to get a fix in place for this, all scans are now being blocked due to this HIGH vulnerability.

**Comment by aaleksandrov** [comment]
Our security scans are screaming and blocking merges

**Comment by Enase** [comment]
Consider making your internal package public to tap into the community's expertise for faster issue resolution.

**Comment by Mmarzex** [comment]
This is resolved with framework version 3.37.
